cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/kafkaassigner/KafkaAssignerEvenRackAwareGoal.java

```diff
@@ -33,12 +33,58 @@
 
 public class KafkaAssignerEvenRackAwareGoal implements Goal {
   private static final Logger LOG = LoggerFactory.getLogger(KafkaAssignerEvenRackAwareGoal.class);
+  private final Map<Integer, SortedSet<BrokerReplicaCount>> _healthyBrokerReplicaCountByPosition;
+  private final Map<Integer, Map<Integer, Integer>> _numExcludedReplicasByPositionInBroker;
```

Comment:
Is it necessary to have a separate map? Can we directly add the excluded replica counts to _healthyBrokerReplicaCountByPosition?